include/wil/stl.h

```diff
@@ -187,6 +187,14 @@ namespace wil
 
     using zstring_view = basic_zstring_view<char>;
     using zwstring_view = basic_zstring_view<wchar_t>;
+
+    constexpr zstring_view operator "" _zv(const char* str, std::size_t len) noexcept {

```

Comment:
IMO - we probably want to put these in an `inline namespace` (e.g. `literals` like the STL) so that folks don't need to `using namespace wil;` and pull in _everything_

Reply:
That's a really good point.  `using namespace wil::literals;` is fine.  `using namespace wil;` is recommended against in most cases.

Reply:
If we want to go _really_ crazy, we could even do `namespace wil::inline namespace literals::inline namespace string_view_literals::operator""_zv` (okay, yes, I know that's not valid syntax). I don't think that's entirely necessary, but would help if WIL has multiple `operator""` definitions and `_zv` conflicts with another library/definition.

Reply:
created an inline namespace.

Reply:
something is off with using "using namespace wil::literals" , the current build has failed. Rechecking

Reply:
If you haven't figured it out already, it's because the definition of `zstring_view` - and therefore the existence of the namespace - is guarded by `#if _HAS_CXX17`